app/views/data_services/show.html.erb

```diff
@@ -1,12 +1,26 @@
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-1" data-cy="data-service-<%= @data_service.data_resource.title %>-name"><%= @data_service.data_resource.title %></h1>
 
-<p class="govuk-body" data-cy="data-service-<%= @data_service.name %>-description"><%= simple_format(@data_service.description) %></p>
+<p class="govuk-body" data-cy="data-service-<%= @data_service.data_resource.title %>-description"><%= simple_format(@data_service.data_resource.description) %></p>
 <dl>
-    <dt class="govuk-heading-m govuk-!-margin-bottom-2">Endpoint</dt>
-    <dd class="govuk-body govuk-!-margin-left-0" data-cy="data-service-<%= @data_service.name %>-endpoint-url"><a class="govuk-link" href="<%= @data_service.url %>"><%= @data_service.url %></a></dd>
-    <dt class="govuk-heading-m govuk-!-margin-bottom-2">Documentation</dt>
-    <dd class="govuk-body govuk-!-margin-left-0" data-cy="data-service-<%= @data_service.name %>-documentation-url"><a class="govuk-link" href="<%= @data_service.documentation_url %>"><%= @data_service.documentation_url %></a></dd>
-    <dt class="govuk-heading-m govuk-!-margin-bottom-2">Contact</dt>
-    <dd class="govuk-body govuk-!-margin-left-0" data-cy="data-service-<%= @data_service.name %>-contact"><a class="govuk-link" href="<%= @data_service.contact %>"><%= @data_service.contact %></a></dd>
+    <%= render 'data_resource_link_detail', field_name: 'Endpoint URL', field_value: @data_service.endpoint_url %>
+    <%= render 'data_resource_link_detail', field_name: 'Endpoint description', field_value: @data_service.endpoint_description %>
+    <%= render 'data_resource_detail', field_name: 'Contact name', field_value: @data_service.data_resource.contact_name %>
+    <%= render 'data_resource_detail', field_name: 'Contact email', field_value: mail_to(@data_service.data_resource.contact_email) %>
+    <%= render 'data_resource_detail', field_name: 'Issued', field_value: @data_service.data_resource.issued&.strftime('%A, %B %d, %Y') %>
+    <%= render 'data_resource_detail', field_name: 'Created', field_value: @data_service.data_resource.created&.strftime('%A, %B %d, %Y') %>
+    <%= render 'data_resource_detail', field_name: 'Modified', field_value: @data_service.data_resource.modified&.strftime('%A, %B %d, %Y') %>
+    <%= render 'data_resource_detail', field_name: 'Identifier', field_value: @data_service.data_resource.identifier %>
+    <%= render 'data_resource_detail', field_name: 'Service type', field_value: @data_service.service_type %>
+    <%= render 'data_resource_detail', field_name: 'Summary', field_value: @data_service.data_resource.summary %>
+    <%= render 'data_resource_detail', field_name: 'Keywords', field_value: @data_service.data_resource.keywords.join(', ') %>
+    <%= render 'data_resource_link_detail', field_name: 'Themes', field_value: @data_service.data_resource.themes.join(', ') %>
```

Comment:
I think the link detail partial won't work for arrays, because if there's more than one it will just create one big link mushed together. Maybe we need a more generic `link` partial that we can apply to each link in the array, and then pass that into the `data_resource_detail` partial as a block? What do you think?